apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/components/shareEmbedModal/share-view.test.tsx

```
@@ -0,0 +1,373 @@
+  Button: ({
+    children,
+    onClick,
+    className,
+    variant,
+  }: {
+    children: React.ReactNode;
+    onClick: () => void;
+    className?: string;
+    variant?: string;
+  }) => (
+    <button onClick={onClick} className={className} data-variant={variant}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)